operators/multiclusterobservability/pkg/config/resources_map.go

```diff
@@ -0,0 +1,496 @@
+
+// Specifies resources for all components and their respective TShirt sizes.
+var (
+	RBACQueryProxyCPURequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "250m",
+		Default:    "20m",
+		Small:      "250m",
+		Medium:     "500m",
+		Large:      "500m",
+		XLarge:     "1",
+		TwoXLarge:  "1",
+		FourXLarge: "1.5",
+	}
+	RBACQueryProxyMemoryRequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "128Mi",
+		Default:    "100Mi",
+		Small:      "256Mi",
+		Medium:     "512Mi",
+		Large:      "1Gi",
+		XLarge:     "2Gi",
+		TwoXLarge:  "2Gi",
+		FourXLarge: "4Gi",
+	}
+
+	GrafanaCPURequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "4m",
+		Default:    "4m",
+		Small:      "4m",
+		Medium:     "4m",
+		Large:      "4m",
+		XLarge:     "4m",
+		TwoXLarge:  "4m",
+		FourXLarge: "4m",
+	}
+	GrafanaMemoryRequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "100Mi",
+		Default:    "100Mi",
+		Small:      "100Mi",
+		Medium:     "100Mi",
+		Large:      "100Mi",
+		XLarge:     "100Mi",
+		TwoXLarge:  "100Mi",
+		FourXLarge: "100Mi",
+	}
+
+	AlertmanagerCPURequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "250m",
+		Default:    "4m",
+		Small:      "500m",
+		Medium:     "500m",
+		Large:      "1",
+		XLarge:     "1",
+		TwoXLarge:  "1500m",
+		FourXLarge: "2",
+	}
+	AlertmanagerMemoryRequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "128Mi",
+		Default:    "200Mi",
+		Small:      "128Mi",
+		Medium:     "256Mi",
+		Large:      "512Mi",
+		XLarge:     "512Mi",
+		TwoXLarge:  "1Gi",
+		FourXLarge: "1Gi",
+	}
+
+	ObservatoriumAPICPURequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "250m",
+		Default:    "20m",
+		Small:      "250m",
+		Medium:     "500m",
+		Large:      "500m",
+		XLarge:     "1",
+		TwoXLarge:  "1",
+		FourXLarge: "1.5",
+	}
+	ObservatoriumAPIMemoryRequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "128Mi",
+		Default:    "128Mi",
+		Small:      "256Mi",
+		Medium:     "512Mi",
+		Large:      "1Gi",
+		XLarge:     "2Gi",
+		TwoXLarge:  "2Gi",
+		FourXLarge: "4Gi",
+	}
+
+	ThanosQueryFrontendCPURequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "500m",
+		Default:    "100m",
+		Small:      "1",
+		Medium:     "1",
+		Large:      "2",
+		XLarge:     "2",
+		TwoXLarge:  "4",
+		FourXLarge: "4",
+	}
+	ThanosQueryFrontendMemoryRequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "256Mi",
+		Default:    "256Mi",
+		Small:      "512Mi",
+		Medium:     "2Gi",
+		Large:      "5Gi",
+		XLarge:     "8Gi",
+		TwoXLarge:  "10Gi",
+		FourXLarge: "12Gi",
+	}
+
+	MemcachedExporterCPURequest ResourceSizeMap = map[observabilityv1beta2.TShirtSize]string{
+		Minimal:    "20m",
+		Default:    "20m",
+		Small:      "5m",
```

Comment:
It decreases?

Reply:
Sorry I think I mixed it up with something, should all be 5m, not customizing this one.